man/man1/ipa-test.1

```diff
@@ -50,6 +50,9 @@ Silence logging information on test run.
 \fB\-\-no\-color\fP
 Remove ANSI color and styling from output.
 .TP
+\fB\-\-no\-default\-test\-dirs\fP
+Do not include default test directories in test run.
```

Comment:
I think this should read:

"Do not include default test directories in search for tests"